.evergreen/config.yml

```diff
@@ -2002,6 +2004,16 @@ axes:
         variables:
           SETDEFAULTENCODING: "cp1251"
 
+  - id: requireApiVersion
+    display_name: "requireApiVersion"
+    values:
+      - id: "requireApiVersion1"
+        display_name: "requireApiVersion1"
+        tags: [ "requireApiVersion_tag" ]
+        variables:
+          REQUIRE_API_VERSION: "1"
```

Comment:
What is the `REQUIRE_API_VERSION` variable used for? I don't see it used anywhere.

Reply:
We need both variables. REQUIRE_API_VERSION tells drivers-evg-tools to start the cluster with the requireApiVersion parameter. MONGODB_API_VERSION tells the test suite which ServerApi.version to use. When we eventually test version 2, we'll need to use:
```
REQUIRE_API_VERSION: "1"
MONGODB_API_VERSION: "2"
```

Reply:
Ah, I see! That's certainly going to be hard to figure out for a newcomer. Can we add a comment that `REQUIRE_API_VERSION` is used by cluster orchestration scripts in drivers-evergreen-tools?

Reply:
Done.